apps/server/src/modules/school/domain/do/school.ts

```diff
@@ -77,16 +111,12 @@ export interface SchoolProps extends AuthorizableObject {
 	federalState: FederalState;
 	county?: County;
 	purpose?: SchoolPurpose;
-	features: Set<SchoolFeature>;
+	features: SchoolFeature[];
```

Comment:
Auch wenn du die InstanceFeatures abgespaltet hast, würde ich sagen, dass Set der richtige Typ ist. Erstens, weil es korrekt aussagt, dass es keinen Sinn macht, wenn es ein Feature mehrmals gibt. Zweitens, weil es auch funktioniert, wenn versehentlich doch ein Feature doppelt in die Datenbank geschrieben wird. Drittens, weil es einfacher ist Features hinzuzufügen oder wegzunehmen, falls das doch mal nötig sein sollte.